FV3/atmos_cubed_sphere/tools/fv_diagnostics.F90

```diff
@@ -745,6 +745,8 @@ subroutine fv_diag_init(Atm, axes, Time, npx, npy, npz, p_ref)
           !            'Relative Humidity', '%', missing_value=missing_value, range=rhrange )
           idiag%id_delp = register_diag_field ( trim(field), 'delp', axes(1:3), Time,        &
                'pressure thickness', 'pa', missing_value=missing_value )
+          idiag%id_delp_total = register_diag_field ( trim(field), 'delp_total', axes(1:3), Time,        &
+               'pressure thickness including condensate mass', 'pa', missing_value=missing_value )
```

Comment:
It occurs to me that our existing coarse diagnostic for the pressure thickness, named `"delp_coarse"`, does not have condensate mass removed, so perhaps should more aptly be named `"delp_total_coarse"`.   I could rename it in this PR.

The other option is to rename the existing `"delp"` diagnostic to reflect that it has condensate mass removed, but I feel less comfortable doing that since it means a deviation from something longstanding in upstream code.

Reply:
That's correct -- I am proposing to only change the name of the online coarse-grained `delp` diagnostic.  Indeed our Q1 and Q2 computations use data from SHiELD (and actually use data from restart files for `delp`).  This is also only the name identifying the diagnostic in the fortran model.  We are free to name it however we want if/when we output it through the diagnostics table in future simulations.